buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -103,24 +102,19 @@ public void apply(Project project) {
                 "Tests for {} require docker-compose at /usr/local/bin/docker-compose or /usr/bin/docker-compose " +
                     "but none could be found so these will be skipped", project.getPath()
             );
-            disableTaskByType(tasks, getTaskClass("com.carrotsearch.gradle.junit4.RandomizedTestingTask"));
```

Comment:
We do  need to disable `Task` here. This plays a role when docker is not available. `-Dtests.fixture.enabled=false` simulates this, so when the plugin is used it will skipp the tests assuming these can't run because there's no docker.

Reply:
I think you meant `Test`, and yes, I believe you are right. I've fixed this.